myuw/static/vue/future_quarters.js

```diff
@@ -0,0 +1,49 @@
+import {Vue, vueConf} from './base.js';
+import utils from './mixins/utils.js';
+
+import Boilerplate from './components/_templates/boilerplate/boilerplate.vue';
+
+// common components
+import AllCourses from './components/_common/course/all-courses.vue';
+import VisualSchedule from './components/_common/visual_schedule/schedule.vue';
+import Textbooks from './components/_common/textbooks.vue';
+
+// stores
+import textbooks from './vuex/store/textbooks';
+import studSchedule from './vuex/store/stud_schedule';
+import visualSchedule from './vuex/store/visual_schedule';
+
+vueConf.store.registerModule('stud_schedule', studSchedule);
+vueConf.store.registerModule('textbooks', textbooks);
+vueConf.store.registerModule('visual_schedule', visualSchedule);
+
+vueConf.store.commit('addVarToState', {
+  name: 'futureTerm',
+  value: window.futureTerm,  // "year,quarter,[ab]-term"
+});
+vueConf.store.commit('addVarToState', {
+  name: 'futureTermData',
+  value: window.futureTermData,
```

Comment:
Did you want to remove the window.futureTermData and use window.termData on all pages?

Reply:
yeah, removing it. I did a project wide search for it but it didn't turn up